jenkins/scripts/files/run_integration_tests.sh

```diff
@@ -36,6 +37,11 @@ then
   unset UPGRADE_TEST
 fi
 
+if [ "${EPHEMERAL_TEST}" == "null" ]
+then
+  unset EPHEMERAL_TEST
```

Comment:
or rather set to default value `false` as in CAPM3?

Reply:
I followed the same way as `UPGRADE_TEST`  and `NUM_NODES` variable.